run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
This is nice to have, but I dont think it is needed by everyone. Consider removing please.

Reply:
Indeed, can you remove this line?